src/main/java/org/opentripplanner/updater/GraphUpdaterConfigurator.java

```diff
@@ -138,7 +139,8 @@ public static void setupUpdaters(Graph graph, List<GraphUpdater> updaters) {
             updaters.add(new MqttGtfsRealtimeUpdater(configItem));
         }
         for (VehicleParkingUpdaterParameters configItem : config.getVehicleParkingUpdaterParameters()) {
-            updaters.add(new VehicleParkingUpdater(configItem));
+            var source = VehicleParkingDataSourceFactory.create(configItem);
```

Comment:
Should this be moved into the updater like it is in VehicleRentalUpdater?

Reply:
We thought it's better to do this outside of the data source constructor. Changed the vehicle rental to be like that too https://github.com/opentripplanner/OpenTripPlanner/pull/3796/commits/124c25bd6b82204a9840149d01314db8648b1b3a